### DISCUSSION TITLE
need help activating Student Pack benefits, have been approved for 3 days

### DISCUSSION THREAD

**Discussion by roshan-pathak** [community/community#147349] [GitHub Education]
### Hi everyone,

Hi! I saw on another post in this forum that we should reach out if our student pack has not been activated after 3 days. I am approaching 3 days and my application has already been accepted, so just looking for help from the support team.

**Comment by davevad93** [comment]
Hi @roshan-pathak ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @roshan-pathak was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
